html/pfappserver/root/static.alt/src/components/pfCSVParse.vue

```diff
@@ -560,7 +560,7 @@ export default {
     },
     selectValues: {
       handler: function (a, b) {
-        if (JSON.stringify(a) === JSON.stringify(b)) return
+        if (a.length === b.length) return
```

Comment:
I'm not suggesting to remove the `JSON.stringify` comparison, but to do it only if the length are different:

```javascript
if (a.length === b.length) && JSON.stringify(a) === JSON.stringify(b)) return
```

Reply:
Length comparison works as expected